### PR TITLE
Completed: Add closing statement after all tutorials #20

### PR DESCRIPTION
**Pull request by martyna-mindsdb**
Fixes: Add closing statement after all tutorials #20->number from a different repo

## Please describe what changes you made, in as much detail as possible
- added the 'What's Next?' section (marked in red box) at the end of each of the five tutorials (marked in red box)
image

mindsdb

### CODE REVIEW

**Review by Ricram2 — CHANGES_REQUESTED** [review]
Minor fix to be done before accepting the PR :).

File: docs/mindsdb-docs/docs/sql/tutorials/customer-churn.md [hunk 1]
```diff
@@ -122,3 +122,13 @@ AND Dependents='No' AND tenure=1 AND PhoneService='No' AND MultipleLines='No pho
 ```
 
 ![customer churn2](/assets/sql/tutorials/customer_churn/customer_churn2.png)
+
+## What's Next?
+
+Have fun while trying it out yourself!
+
+* Bookmark [MindsDB repository on GitHub](https://github.com/mindsdb/mindsdb).
+* Sign up for a free [MindsDB account](https://cloud.mindsdb.com/?_gl=1*16uh9sn*_ga*ODkzODkzMDc2LjE2NTYwMTgwNjY.*_ga_7LGFPGV6XV*MTY1ODMyNjU0Ni44LjEuMTY1ODMyNjYxMy4w&_ga=2.127186189.1258054943.1658158446-893893076.1656018066).
```
[the remark this answers is not in the document]
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/sql/tutorials/customer-churn.md]: Updated the link.

File: docs/mindsdb-docs/docs/sql/tutorials/home-rentals.md [hunk 2]
```diff
@@ -181,3 +181,13 @@ JOIN mindsdb.home_rentals_model as m limit 100
 | 4774       | 4748            | 3               | 2                   | 1066 | good     | 13             |
 +------------+-----------------+-----------------+---------------------+------+----------+----------------+
 ```
+
+## What's Next?
+
+Have fun while trying it out yourself!
+
+* Bookmark [MindsDB repository on GitHub](https://github.com/mindsdb/mindsdb).
+* Sign up for a free [MindsDB account](https://cloud.mindsdb.com/?_gl=1*16uh9sn*_ga*ODkzODkzMDc2LjE2NTYwMTgwNjY.*_ga_7LGFPGV6XV*MTY1ODMyNjU0Ni44LjEuMTY1ODMyNjYxMy4w&_ga=2.127186189.1258054943.1658158446-893893076.1656018066).
```
[the remark this answers is not in the document]
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/sql/tutorials/home-rentals.md]: Updated the link.

File: docs/mindsdb-docs/docs/sql/tutorials/house-sales-forecasting.md [hunk 3]
```diff
@@ -56,6 +56,12 @@ LIMIT 4;
...
 Now, try changing `type` to unit or `bedrooms` to any number between 1 to 5, and check how the forecast varies. This is because MindsDB recognizes each grouping as being its own different time series.
 
-## Next steps
+## What's Next?
 
-We invite you to try this tutorial with your own time series data. If you have any questions or feedback, be sure to join our slack community or make an issue in the github repository!
+Have fun while trying it out yourself!
+
+* Bookmark [MindsDB repository on GitHub](https://github.com/mindsdb/mindsdb).
+* Sign up for a free [MindsDB account](https://cloud.mindsdb.com/?_gl=1*16uh9sn*_ga*ODkzODkzMDc2LjE2NTYwMTgwNjY.*_ga_7LGFPGV6XV*MTY1ODMyNjU0Ni44LjEuMTY1ODMyNjYxMy4w&_ga=2.127186189.1258054943.1658158446-893893076.1656018066).
```
[the remark this answers is not in the document]
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/sql/tutorials/house-sales-forecasting.md]: Updated the link.

File: docs/mindsdb-docs/docs/sql/tutorials/mindsdb-superset-snowflake.md [hunk 4]
```diff
@@ -174,3 +174,13 @@ The combination of MindsDB and your database covers all the phases of the ML lif
 MindsDB provides easy-to-use predictive models through AI Tables. You can create these predictive models using SQL statements and feeding the input data. Also, you can query them the same way you query a table. The easiest way to get started with Superset is with the free tier for [Preset Cloud](https://preset.io/product/), a hassle-free and fully hosted cloud service for Superset.
 
 We encourage you to try some predictions with your own data, so please sign up for a [free MindsDB cloud account](https://cloud.mindsdb.com/signup) and if you need any help with MindsDB, feel free to ask our [Slack](https://join.slack.com/t/mindsdbcommunity/shared_invite/zt-o8mrmx3l-5ai~5H66s6wlxFfBMVI6wQ) and [Github](https://github.com/mindsdb/mindsdb/discussions) communities.
+
+## What's Next?
+
+Have fun while trying it out yourself!
+
+* Bookmark [MindsDB repository on GitHub](https://github.com/mindsdb/mindsdb).
+* Sign up for a free [MindsDB account](https://cloud.mindsdb.com/?_gl=1*16uh9sn*_ga*ODkzODkzMDc2LjE2NTYwMTgwNjY.*_ga_7LGFPGV6XV*MTY1ODMyNjU0Ni44LjEuMTY1ODMyNjYxMy4w&_ga=2.127186189.1258054943.1658158446-893893076.1656018066).
```
[the remark this answers is not in the document]
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/sql/tutorials/mindsdb-superset-snowflake.md]: Updated the link.

File: docs/mindsdb-docs/docs/sql/tutorials/process-quality.md [hunk 5]
```diff
@@ -158,8 +158,12 @@ As you can see below, the predictor has made multiple predictions for each data
...
 
-You are now done with the tutorial! 🎉
+## What's Next?
 
-Please feel free to try it yourself. Sign up for a [free MindsDB account](https://cloud.mindsdb.com/signup?utm_medium=community&utm_source=ext.%20blogs&utm_campaign=blog-manufacturing-process-quality) to get up and running in 5 minutes, and if you need any help, feel free to ask in [Slack](https://join.slack.com/t/mindsdbcommunity/shared_invite/zt-o8mrmx3l-5ai~5H66s6wlxFfBMVI6wQ) or [Github](https://github.com/mindsdb/mindsdb/discussions).
+Have fun while trying it out yourself!
 
-For more tutorials like this, check out [MindsDB documentation](https://docs.mindsdb.com/).
+* Bookmark [MindsDB repository on GitHub](https://github.com/mindsdb/mindsdb).
+* Sign up for a free [MindsDB account](https://cloud.mindsdb.com/?_gl=1*16uh9sn*_ga*ODkzODkzMDc2LjE2NTYwMTgwNjY.*_ga_7LGFPGV6XV*MTY1ODMyNjU0Ni44LjEuMTY1ODMyNjYxMy4w&_ga=2.127186189.1258054943.1658158446-893893076.1656018066).
```
[the remark this answers is not in the document]
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/sql/tutorials/process-quality.md]: Updated the link.

**Comment by ZoranPandovski** [comment]
recheckcla

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**